docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -44,6 +67,365 @@ To utilize this store you will import the `CART_STORE_KEY` in any module referen
 const { CART_STORE_KEY } = window.wc.wcBlocksData;
 ```
 
+## Actions
+
+### setCartData
+
+This action is used to set the cart data in the store.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartData: Object`: The new cart data.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( setCartData( newCartData ) );
+```
+
+### setErrorData
+
+This action is used to set the error data in the store.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `errorData: Object`: The new error data.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( setErrorData( newErrorData ) );
+```
+
+### receiveCartContents
+
+This action returns an action object used in updating the store with the provided cart. It omits the customer addresses so that only updates to cart items and totals are received.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartContents: Object`: The new cart contents.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
```

Comment:
Similar to the _Parameters_, what are your thoughts on documenting the inner structure of the objects within the _Returns_ sections? 

Here are two examples of other docs:

https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/data-store/collections.md?plain=1#L52

https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/data-store/collections.md?plain=1#L69-L74